units/pod_termination.go

```diff
@@ -202,8 +203,14 @@ func (j *podTerminationJob) populateIfUnset(ctx context.Context) error {
 // tasks in the dispatcher as needing re-allocation.
 func (j *podTerminationJob) fixStrandedTasks(ctx context.Context) error {
 	if j.pod.RunningTask != "" {
+		// A stranded task will need to be re-allocated to ensure that it
+		// dispatches to a new pod after this one is terminated.
+		if err := task.MarkManyContainerDeallocated([]string{j.pod.RunningTask}); err != nil {
```

Comment:
`MarkManyContainerDeallocated` is a bit confusing since I'm assuming there are going to be additional cases potentially where we'd just want to de-allocate a single container task.  Can we rename to something like `MarkTasksAsContainerDeallocated`?

Reply:
Sounds good, changed.